pkg/util/storage.go

```diff
@@ -0,0 +1,36 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019 Datadog, Inc.
+
+package util
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+)
+
+func getFileForKey(key string) string {
+	cleanedKey := strings.Replace(key, ":", "_", -1)
```

Comment:
Why does the key contains a `:` and why replace only the last one ?

Reply:
They key is the check_id, which is seperated by `:`, and the `-1` means that it replaces all the occurences.